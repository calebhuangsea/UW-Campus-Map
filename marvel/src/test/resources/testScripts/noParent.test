# Find a path where parent node doesn't exist

LoadGraph noParent noParent.csv

FindPath noParent n2 n1