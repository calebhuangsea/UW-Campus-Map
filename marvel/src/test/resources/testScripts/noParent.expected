# Find a path where parent node doesn't exist

loaded graph noParent

unknown: n2